tools/indexer-api-caller/caller.go

```diff
@@ -16,10 +16,11 @@ import (
 
 var (
 	repoDir   = flag.String("repo", "", "repo directory")
+	repoDir2  = flag.String("repo2", "", "specify another directory to compare file hashes to the first")
```

Comment:
What's this used for? Was this just for debugging this issue locally?

Reply:
It helps debugging issues locally, also can be used to get an actual exact number of file matches once you identified what the estimate is with the API. (e.g. you download the repo@tag that determineversion API returns, and run it against what you initially are querying with)

Reply:
SG. Let's merge and see if we can get this into this week's release?